frontend/src/metabase/visualizations/visualizations/Table.jsx

```diff
@@ -432,3 +441,5 @@ export default class Table extends Component {
     );
   }
 }
+
+export default _.compose(connect(mapStateToProps))(Table);
```

Comment:
nit: since we're not wrapping Table inside more than 1 higher-order function yet, maybe it's better to leave out `compose`

```suggestion
export default connect(mapStateToProps)(Table);
```

Reply:
Makes sense. I was copying the style here, which could also do with changing:
https://github.com/metabase/metabase/blob/22521-metadata-tab-details-view-only/enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditTable.jsx#L98